custom-conventional-commits-accepted.json

```diff
@@ -0,0 +1,14 @@
+[
```

Comment:
this is only for local testing purpose, the team will provide their own custom-cc in a parameter in the new gh action